roles/ipasmartcard_client/tasks/main.yml

```diff
@@ -0,0 +1,173 @@
+---
+# tasks file for ipasmartcard_client role
+
+- name: Uninstall smartcard client
+  fail: msg="Uninstalling smartcard for IPA is not supported"
```

Comment:
Some modules are using FQCN, some are not. I suggest using FQCN for all built-in modules.

The ones I found not using FQCN are `fail`, `package`, `command`, `set_fact` and `script`.